plugins/source/azure/docs/tables/azure_authorization_role_definitions.md

```diff
@@ -14,11 +14,11 @@ The primary key for this table is **id**.
 |_cq_id|UUID|
 |_cq_parent_id|UUID|
 |subscription_id|String|
+|properties_assignable_scopes|JSON|
```

Comment:
Can you explain the context behind pre-fixing `properties`? Since this is a pattern the Azure SDK uses, and we unwrap them, I'm not sure we should add the prefix to column names (nor anyone complained that we don't)

Reply:
done